clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java

```diff
@@ -183,91 +151,67 @@ public FileOutStream getOutStream(TachyonURI path) throws IOException, InvalidPa
    * @param path the Tachyon path of the file
    * @param options the set of options specific to this operation
    * @return an output stream to write the file
-   * @throws InvalidPathException if the provided path is invalid
-   * @throws FileAlreadyExistException if the file being written to already exists
-   * @throws BlockInfoException if the provided block size is invalid
-   * @throws IOException if the file already exists or if the stream cannot be opened
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws TachyonException if a Tachyon exception occurs
    */
-  public FileOutStream getOutStream(TachyonURI path, ClientOptions options) throws IOException,
-      InvalidPathException, FileAlreadyExistException, BlockInfoException {
-    long fileId = super.create(path, options.getBlockSize(), true, options.getTTL());
+  public FileOutStream getOutStream(TachyonURI path, OutStreamOptions options) throws IOException,
+      TachyonException {
+    CreateOptions createOptions =
+        (new CreateOptions.Builder(new TachyonConf())).setBlockSize(options.getBlockSize())
+            .setRecursive(true).setTTL(options.getTTL()).build();
```

Comment:
Using true is consistent with the previous logic and I probably does not make sense to ever use false here. (The OutStreamOptions does not contain the recursive flag).